docs/data/material/getting-started/templates/landing-page/getLPTheme.tsx

```diff
@@ -268,11 +268,11 @@ export default function getLPTheme(mode: PaletteMode): ThemeOptions {
       },
       MuiToggleButton: {
```

Comment:
@danilo-leal any idea why the styles here on `MuiToggleButtonGroup` label aren't working? On light is kinda ok, but the contrast is lame on dark theme.

<img width="359" alt="Screenshot 2024-02-01 at 08 51 18" src="https://github.com/mui/material-ui/assets/37222944/9ad4081e-aedd-4d3e-834a-b6b3ebba1c91">